packages/falso/src/lib/json.ts

```diff
@@ -88,5 +96,9 @@ export function randJSON<Options extends RandomJSONOptions = never>(
     return generatedObject;
   };
 
-  return fake(factory, options);
+  return fake(factory, options, { uniqueComparer: checkUnique });
+}
+
+function checkUnique(item: object, items: object[]): boolean {
+  return items.some((i) => JSON.stringify(i) === JSON.stringify(item));
```

Comment:
Isn't this checking the opposite? Don't you need all jsons to be different?
Looking at this as a reference:
![image](https://user-images.githubusercontent.com/17194830/191011268-a3a76d44-54bd-48d1-a0a3-7218294c8689.png)